dev/io.openliberty.org.jboss.resteasy.common/src/io/openliberty/restfulWS/config/ConfigImpl.java

```diff
@@ -35,8 +35,20 @@
             });
         }
         for (Object key : System.getProperties().keySet()) {
+            if (key.equals("resteasy.unwrapped.exceptions")) {
+                
+            }
```

Comment:
This doesn't do anything - maybe a path you started down, but abandoned? I think it should probably be removed.

Reply:
thanks